src/jit/lowerxarch.cpp

```diff
@@ -4471,6 +4497,50 @@ bool Lowering::ExcludeNonByteableRegisters(GenTree* tree)
             return false;
         }
     }
+#ifdef FEATURE_SIMD
+    else if (tree->OperGet() == GT_SIMD)
+    {
+        GenTreeSIMD* simdNode = tree->AsSIMD();
+        var_types    baseType = simdNode->gtSIMDBaseType;
```

Comment:
>var_types    baseType = simdNode->gtSIMDBaseType; [](start = 8, length = 49)

Minor: this variable definition could be moved closer to the point of its use in the block starting at line 4518